lib/media/media_source_engine.js

```diff
@@ -72,6 +76,8 @@ shaka.media.MediaSourceEngine = class {
                     number, ?number)} */
     this.onMetadata_ = onMetadata || onMetadataNoOp;
 
+    this.lcevcDil_ = lcevcDil;
```

Comment:
Please document the type here, and please clean it up in doDestroy_().

Reply:
This is rectified.